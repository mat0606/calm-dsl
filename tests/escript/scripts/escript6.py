# python2;failure
pnt
