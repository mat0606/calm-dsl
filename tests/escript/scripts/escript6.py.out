Error at line 3> name 'pnt' is not defined
